src/imitation/data/buffer.py

```diff
@@ -4,10 +4,10 @@
 import numpy as np
 from stable_baselines3.common import vec_env
 
-from imitation.data import datasets, types
+from imitation.data import types
 
 
-class Buffer(datasets.Dataset[Dict[str, np.ndarray]]):
+class Buffer:
```

Comment:
+1 on deleting datasets (always nice to have less code!) -- but it seems nice to highlight that these classes do implement a common interface. e.g. both have `from_data`, `store`, `sample` differing only in types. Perhaps we can make a new (generic) abstract class for this?

Reply:
I don't think this effort is worthwhile right now, especially since among the two classes we are only using `ReplayBuffer` and in only one place.

Reply:
OK, I don't feel strongly about this.